hack/virtc/main.go

```diff
@@ -250,6 +250,63 @@ func buildPodConfig(context *cli.Context) (vc.PodConfig, error) {
 	return podConfig, nil
 }
 
+// checkRequiredPodArgs checks to ensure the required command-line
+// arguments have been specified for the pod sub-command specified by
+// the context argument.
+func checkRequiredPodArgs(context *cli.Context) error {
+	if context == nil {
+		return nil
+	}
+
+	// sub-sub-command name
+	name := context.Command.Name
+
+	switch name {
+	case "create":
+		fallthrough
+	case "list":
+		fallthrough
+	case "run":
+		return nil
+	}
+
+	id := context.String("id")
+	if id != "" {
+		return nil
```

Comment:
You should test id == "" and return the error here, rather than return nil from here. It would be more consistent with the way we write functions usually, don't you think ?